Sources/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -75,6 +78,10 @@ public extension MiniAppMessageDelegate {
     func downloadFile(fileName: String, url: String, headers: DownloadHeaders, completionHandler: @escaping (Result<String, MASDKError>) -> Void) {
         completionHandler(.failure(.failedToConformToProtocol))
     }
+
```

Comment:
You forgot to implement 'closeMiniApp' interface

Reply:
Thank you for finding this, my bad I have updated in the latest commit.